src/components/NavBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { AppBar, Toolbar, Typography, Chip } from '@material-ui/core/';
+
+const drawerWidth = 240;
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    appBar: {
+      width: `calc(100% - ${drawerWidth}px)`,
+      marginLeft: drawerWidth,
+      backgroundColor:'#dd7200'
+    },
+    network: {
+
+    }
+  }),
+);
+
+export default function SideBar() {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <CssBaseline />
```

Comment:
This should probably be directly in the `App.tsx`, right?

Reply:
Agreed, fix in a later commit